app/assets/scripts/a11n/signin.js

```diff
@@ -58,31 +114,62 @@ function SignIn() {
     return errors;
   }, []);
 
-  const onSubmit = useCallback(
-    async (values, { setSubmitting }) => {
-      const processToast = createProcessToast('Signing in. Please wait.');
+  const qrString = `otpauth://totp/AWSCognito:${user?.username}?secret=${mfaCode}`;
+
+  const handleOtpCodeInputChange = React.useCallback((e) => {
+    setOtpCode(e.target.value);
+  }, []);
+
+  const handleVerifyOtpSubmission = React.useCallback(async () => {
+    toastRef.current.update('Verifying OTP, Please wait...');
```

Comment:
Can this just be a button behavior rather than a toast? Something like "Verifying..." when a button is clicked.

Reply:
Hmm.. that can be done! I'll try it out.

Reply:
@xhagrg should we implement these toast/button design decisions with APT-wide consistency, like @deborahUAH suggested? I don't remember having button-based verifications elsewhere, but perhaps we do...